.github/workflows/package-release.yml

```diff
@@ -0,0 +1,110 @@
+name: "Package Release"
+run-name: "Package Release ${{ github.ref_name }}"
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build-pecl:
+    name: "Create PECL package"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v4"
+        with:
+          # Manually specify a ref. When actions/checkout is run for a tag without a ref, it looks up the underlying
+          # commit and specifically fetches this to the refs/tags/<tag> ref, which denies us access to the tag message
+          ref: ${{ github.ref }}
+          submodules: true
+
+      - name: "Build Driver"
+        uses: ./.github/workflows/linux/build
+        with:
+          version: "8.3"
+
+      - name: "Write changelog file for packaging"
+        run: git tag -l ${{ github.ref_name }} --format='%(contents)' > changelog
+
+      # This will fill in the release notes from the previously generated changelog
+      - name: "Build package.xml"
+        run: "make package.xml RELEASE_NOTES_FILE=$(pwd)/changelog"
```

Comment:
Why is `$(pwd)` used here, but `./` earlier? Wouldn't all of these steps run in the same directory if you're not explicitly changing paths?

Reply:
The filename is used in a PHP file that is in a different directory. I could either assume that it's always a path relative to the root directory and use `__DIR__`, or ensure that we're passing an absolute path. I could be missing something, but I figured an absolute path would be the safest way to avoid issues.

Reply:
Ah, I overlooked that this was being passed into a script. I concur that an absolute path is safest.